dev_tools/nyaml2nxdl/nyaml2nxdl.py

```diff
@@ -175,17 +178,17 @@ def split_name_and_extension(file_name):
     help="The path to the XML or YAML input data file to read and create \
 a YAML or XML file from, respectively.",
 )
-@click.option(
-    "--append",
-    help="Parse xml file and append to base class, given that the xml file has same name \
-of an existing base class",
-)
+# @click.option(
+#     "--append",
+#     help="Parse xml file and append to base class, given that the xml file has same name \
+# of an existing base class",
+# )
 @click.option(
     "--check-consistency",
     is_flag=True,
     default=False,
     help=(
-        "Check wether yaml or nxdl has followed general rules of scema or not"
+        "Check if yaml or nxdl has followed general rules of whether schema or not"
```

Comment:
"... corresponding schema.~~or not~~"

Reply:
Too many `check` words here.  Rephrase.  Be concise.